tests/testthat/_snaps/pivot-wide.md

```diff
@@ -57,7 +57,7 @@
     Output
       <error/rlang_error>
       Error in `build_wider_spec()`:
-      ! `values_from` must select at least one column.
+      ! Must select at least one item.
```

Comment:
These used to report `names_from` vs `values_from` as the place where the problem came from, which was nice

Reply:
Issue: https://github.com/r-lib/tidyselect/issues/327